Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -8,24 +8,12 @@
 enum ExpressionParser {
     static func parse(from input: String) -> Formula {
         let components = componentsByOperators(from: input)
-        let operands = CalculatorItemQueue<Double>(enqueueBox: [])
-        let operators = CalculatorItemQueue<Operator>(enqueueBox: [])
-        let inputArray = input.map { $0 }
-        var formula = Formula(operands: operands, operators: operators)
+        let operandsComponents = components.compactMap { Double($0) }
+        let operatorsComponents = input.map { $0 }.compactMap { Operator(rawValue: $0) }
```

Comment:
개인적으로 고차함수를 체이닝할 때 개행을 추가하면 가독성이 좋아진다고 생각합니다. 😊

```suggestion
        let operatorsComponents = input.map { $0 }
            .compactMap { Operator(rawValue: $0) }
```